rpm/sailfish-setup.spec

```diff
@@ -33,14 +33,16 @@ if ! getent passwd sailfish-actdead >/dev/null ; then
     useradd -r -g sailfish-actdead -d / -s /sbin/nologin sailfish-actdead || :
 fi
 
+groupadd -rf oneshot || :
+
 # TODO : This should be moved to the first boot (JB#48049)
 # Make sure that device owner exists.
 groupadd -fg 100000 nemo || :
 if ! getent passwd nemo >/dev/null ; then
     # Requirements - source component in brackets:
     # video for display (setup), input input devices (systemd), audio for itself (setup)
     # users (setup)
-    useradd -g nemo -G "video,input,audio,users" -u 100000 -m nemo || :
+    useradd -g nemo -G "video,input,audio,users,oneshot" -u 100000 -m nemo || :
```

Comment:
Discussed with Simo. Cleaner / safer to add this as a separate supplementary group add.